app/src/NativeBase/Components/Forms/DatePicker.tsx

```diff
@@ -0,0 +1,195 @@
+/**
+ * @file DatePicker.
+ */
+
+import React, { FC, useState } from 'react'
+import CalendarPicker from 'react-native-calendar-picker'
+import { Pressable, Dimensions } from 'react-native'
+import { VStack, Box, Text, Input, useColorMode, Icon } from 'native-base'
+import dayjs from 'dayjs'
+import StaTheme from '../../Theme/StaTheme'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+
+interface DatePickerProps {
+  description: string
+  maximumDate?: Date
+  minimumDate?: Date
+  onChange: (date: Date) => void
+  value: Date
+}
+
+const DatePicker: FC<DatePickerProps> = ({
+  description,
+  maximumDate,
+  minimumDate,
+  onChange,
+  value,
+}) => {
+  const [pickerActive, setPickerActive] = useState(false)
+  const [selectedStartDate, setSelectedStartDate] = useState(null)
```

Comment:
I think this needs an explicit type set here i.e. `useState<Date | null>(null)`, otherwise my Typescript linter is showing me an error on line 36 where it's being set to a `Date`.  

(With the default value being set here as `null` Typescript just assumes it'll always be `null` unless we explicitly tell it otherwise.)